spec/models/validations/soft_validations_spec.rb

```diff
@@ -84,7 +75,8 @@
 
   describe "retirement soft validations" do
     before do
-      record.update!(age1:, ecstat1:)
+      record.age1 = age1
+      record.ecstat1 = ecstat1
```

Comment:
really unimportant, here, maybe worthwhile below, you could use `update_attributes(age1:, ecstat1:)`